spikewidgets/widgets/basewidget.py

```diff
@@ -28,7 +28,7 @@ def get_name(self):
 
 class BaseMultiWidget:
     def __init__(self, figure=None, ax=None):
-        if figure is None and ax is None:
+        if (figure is None) and (ax is None):
```

Comment:
parenthesis are not required in Python ehehe

Reply:
Sure, personal preference here :)

Reply:
okok